src/EditorFeatures/TestUtilities/Traits.cs

```diff
@@ -189,6 +189,7 @@ public static class Features
             public const string XmlTagCompletion = nameof(XmlTagCompletion);
             public const string CodeActionsAddOverload = "CodeActions.AddOverloads";
             public const string CodeActionsAddNew = "CodeActions.AddNew";
+            public const string CodeActionsAddParenthesis = nameof(CodeActionsAddParenthesis);
```

Comment:
Please keep this list sorted.

Reply:
this sholud be: ```public const string CodeActionsAddParenthesis = "CodeActions.AddParenthesis";``` so that the tests show up properly in the xunit runner.

Reply:
This should be CodeActionsAddParenthesisAroundXXX to match the name of your type.